website/i18n/ru/docusaurus-plugin-content-docs/current/usage/methods.md

```diff
@@ -112,7 +112,78 @@ class SomeApiService::Posts::Create < SomeApiService::Base
 end
 ```
 
-## Сокращения для методов
+## Группа из нескольких методов
+
+Собрать в одну группу выполнение несколько методов можно при помощи метода `stage`.
+
+:::info
+
+Использование опции `position` для `make` будет сортировать только внутри `stage`.
+
+:::
+
+```ruby
+stage do
+  make :create_user!
+  make :create_blog_for_user!
+  make :create_post_for_user_blog!
+end
+```
+
+### Оборачивание
+
+Группу методов, находящийхся в `stage` можно обернуть во что-то.
+Например, это может быть `ActiveRecord::Base.transaction` от Rails.
+
+:::caution
+
+Вызов метода `wrap_in` должен находиться над вызовом метода `make`.
+
+:::
+
+```ruby
+stage do
+  # highlight-next-line
+  wrap_in ->(methods:) { ActiveRecord::Base.transaction { methods } }
+  
+  make :create_user!
+  make :create_blog_for_user!
+  make :create_post_for_user_blog!
+end
+```
+
+### Откат
+
+Если в одном из методов в группе или в `wrap_in` возникло исключение, то это можно обработать при помощи метода `rollback`.
```

Comment:
If an exception occurs in one of the methods in the group or in `wrap_in`, this can be handled using the `rollback` method.